docs/001_develop/03_client-capabilities/008_interaction/combobox.mdx

```diff
@@ -0,0 +1,374 @@
+
+    Declaration
+    ```html
+    <rapid-combobox>
+      <rapid-option value="s">Small</rapid-option>
+      <rapid-option value="m">Medium</rapid-option>
+      <rapid-option value="l">Large</rapid-option>
+    </rapid-combobox>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-combobox @change=${x => x.selectChanged()}>
+          <rapid-option value="s">Small</rapid-option>
+          <rapid-option value="m">Medium</rapid-option>
+          <rapid-option value="l">Large</rapid-option>
+        </rapid-combobox>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      selectChanged() {
+        alert('Select Changed')
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+  Declaration
+  ```html
+  <rapid-combobox>
+    <rapid-option value="s">Small</rapid-option>
+    <rapid-option value="m">Medium</rapid-option>
+    <rapid-option value="l">Large</rapid-option>
+  </rapid-combobox>
+  ```
+  Usage
+  ```typescript
+  export function MyComponent() {
+
+    const [outputValue, setOutputValue] = useState('');
+
+    const onChange = (event) => {
+      setOutputValue(event.target.value);
+      console.log(this.outputValue);
+    }
+
+    return (
+      <rapid-combobox onChange={onChange}>
+        <rapid-option value="s">Small</rapid-option>
+        <rapid-option value="m">Medium</rapid-option>
+        <rapid-option value="l">Large</rapid-option>
+      </rapid-combobox>
+    );
+  }
+  ```
+</TabItem>
+  <TabItem value="angular">
+
+  Declaration
+  ```html
+  <rapid-combobox>
+    <rapid-option value="s">Small</rapid-option>
+    <rapid-option value="m">Medium</rapid-option>
+    <rapid-option value="l">Large</rapid-option>
+  </rapid-combobox>
+  ```
+
+  Usage
+  ```typescript
+  import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+  import { FormsModule } from '@angular/forms';
+  @Component({
+    selector: 'my-root',
+    template: `
+          <rapid-combobox (change)="onChange($event)">
+            <rapid-option value="s">Small</rapid-option>
+            <rapid-option value="m">Medium</rapid-option>
+            <rapid-option value="l">Large</rapid-option>
+          </rapid-combobox>
+        `,
+    standalone: true,
+    schemas: [CUSTOM_ELEMENTS_SCHEMA],
+    imports: [FormsModule],
+  })
+  export class AppComponent {
+    onChange(event: Event) {
+      const target = event.target as HTMLInputElement;
+      console.log(target.value);
+    }
+  }
+  ```
+</TabItem>
+</Tabs>
+
+## API
+
+Property and attribute binding examples for Genesis Component syntax. Closing tag omitted.
+
+<table>
+  <thead><tr><th>Name</th><th>Type</th><th>Description</th><th>Example</th></tr></thead>
+  <tbody>
+    <tr>
+      <td>autocomplete</td>
+      <td><code>"inline" | "list" | "both" | "none"</code></td>
+      <td>Defines how autocompletion works in the component. **Default:** `none`</td>
```

Comment:
If you wrap the explanation in the tip in a div and give it an id you can link to it using an anchor tag here